sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 4.2.0-beta.5 (Unreleased)
```

Comment:
Update the release date as well.

Reply:
I prefer to update the release date until I have the exact date, sometimes things get delayed like when waiting for some dependencies to release.